README.md

```diff
@@ -44,7 +44,7 @@ To get a local copy up and running follow these simple example steps.
 
 ## Screenshot of the project
 
-<img width="1268" alt="Screen Shot 2022-03-23 at 20 05 47" src="./src/leaderboard.png">
```

Comment:
- [ ] Please add a meaningful project description to your README file. A project description is a high-level overview of why you're doing a project. The document explains the objectives of a project and its essential qualities. Think of it as an elevator pitch that focuses on what and why without going into how.

- I would also recommend to add the following:


```

## Show your support

Give a ⭐️ if you like this project!

## Acknowledgments

- Microverse
```